go/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# FIXME(typhoonzero): Download glide into cmake build temprary GOPATH
+if(EXISTS $ENV{GOPATH}/bin/glide)
+  set(GLIDE "$ENV{GOPATH}/bin/glide")
+else()
+  message(FATAL_ERROR "no glide executeble found: $ENV{GOPATH}/bin/glide")
```

Comment:
Maybe instead of fail, we can do something like `execute_process(env GOPATH ${PADDLE_GO_PATH} ${CMAKE_Go_COMPILER} get github.com/Masterminds/glide   (forgot the exact cmake command to temporarily set environment variable).